stdlib/flux_test.go

```diff
@@ -33,6 +33,7 @@ var skip = map[string]string{
 	"yield":                       "yield requires special test case (https://github.com/influxdata/flux/issues/535)",
 	"task_per_line":               "join produces inconsistent/racy results when table schemas do not match (https://github.com/influxdata/flux/issues/855)",
 	"rowfn_with_import":           "imported libraries are not visible in user-defined functions (https://github.com/influxdata/flux/issues/1000)",
+	"string_trim":                 "cannot reference a package function from within a row function",
```

Comment:
Is this a future fix? I thought the local identifier scope and the package issue were the same one.

Reply:
yeah I wrote that with the intention of getting it to pass.  This PR is a half-solution to get object identifiers visible and working for the front-end team